docs-user/guide-startup-shutdown.md

```diff
@@ -5,7 +5,12 @@
 Before starting, make sure the profiler's popup is already present. Otherwise
 please go to https://profiler.firefox.com to add it first to your Firefox.
 
-1. Start your Firefox with the environment variable `MOZ_PROFILER_STARTUP=1` set. This way, the profiler is started as early as possible during startup.
+1. Start your Firefox with the environment variable `MOZ_PROFILER_STARTUP=1` set. This way, the profiler is started as early as possible during startup. 
```

Comment:
super nit: Looks like there are trailing spaces in line 8 and 10. I don't know why our linters don't complain, but probably they don't check the markdown files. Could you please remove them?

Reply:
Ah they check the markdown files, but docs-user is explicitely ignored in `.prettierignore`. I believe initially this was just ot ignore the JS files there, because prettier was checking only that. I guess we could update this `.prettierignore `.

Reply:
I'm fixing it in #5358.